### PR TITLE
translate two release note on 2014/11/13 (zh_cn)

### PR DESCRIPTION
**Pull request by cassiuschen**
translate two release notes posted on 2014/11/13.
Related #1074

### CODE REVIEW

File: zh_cn/news/_posts/2014-11-13-ruby-1-9-3-p551-is-released.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+date: 2014-11-13 12:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布 Ruby 1.9.3-p551 发布了。
+
+本次发布包括了一个针对 REXML 拒绝服务攻击（DoS）漏洞的安全修复。
+
+下述日志中包含相似内容：
+[预发布](https://www.ruby-lang.org/en/news/2014/10/27/ruby-1-9-3-p550-is-released/) 中的 [漏洞修复](https://www.ruby-lang.org/en/news/2014/10/27/rexml-dos-cve-2014-8080/),
```
Comment (JuanitoFatas) [inline zh_cn/news/_posts/2014-11-13-ruby-1-9-3-p551-is-released.md]: > [预发布](https://www.ruby-lang.org/en/news/2014/10/27/ruby-1-9-3-p550-is-released/)
if no zh_cn available, please keep en version as is.

> 中的 [漏洞修复]

no spaces between Chinese characters.

> (https://www.ruby-lang.org/en/news/2014/10/27/rexml-dos-cve-2014-8080/),

Use Chinese punctuations.

**Comment by cassiuschen** [comment]
@JuanitoFatas Fixed.

**Comment by stomar** [comment]
Please squash into single commit before merging.

**Comment by JuanitoFatas** [comment]
> Please squash into single commit before merging, in cases like this with minor fixes.

Noted. Thanks.

**Comment by cassiuschen** [comment]
@stomar @JuanitoFatas Done!

**Comment by JuanitoFatas** [comment]
Thanks! Merged manually as 2577827.

File: zh_cn/news/_posts/2014-11-13-ruby-2-0-0-p598-is-released.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+translator: "cassiuschen"
+date: 2014-11-13 12:00:00 +0000
+lang: zh_cn
+---
+
+我们很高兴地宣布 Ruby 2.0.0-p598 发布了。
+
+本次发布包括了一个针对 REXML 拒绝服务攻击（DoS）漏洞的安全修复。
+下述日志中包含相似内容：
+[预发布](https://www.ruby-lang.org/en/news/2014/10/27/ruby-1-9-3-p550-is-released/)中的[漏洞修复](https://www.ruby-lang.org/en/news/2014/10/27/rexml-dos-cve-2014-8080/),
```
Comment (stomar) [inline zh_cn/news/_posts/2014-11-13-ruby-2-0-0-p598-is-released.md]: s/ruby-1-9-3-p550/ruby-2-0-0-p594/ (fixed)

Comment (JuanitoFatas) [inline zh_cn/news/_posts/2014-11-13-ruby-2-0-0-p598-is-released.md]: Ah...Thanks!

**Pull request closed without merging**